ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -590,15 +616,17 @@ private[state] object Conversions {
           buildStatus(
             Code.INVALID_ARGUMENT,
             s"Party not known on ledger: Submitting party '${rejection.getSubmitterParty}' not known",
+            Map("submitter_party" -> rejection.getSubmitterParty),
           )
         )
       case DamlTransactionRejectionEntry.ReasonCase.PARTIES_NOT_KNOWN_ON_LEDGER =>
         val rejection = entry.getPartiesNotKnownOnLedger
+        val parties = rejection.getPartiesList.asScala.mkString("[", ",", "]")
```

Comment:
I think we might want to JSON-encode this so strings are correctly quoted. See `JsonStringSerializer` in _contextualized-logging_ for an example of how to do this.